pl.lproj/Localizable.strings

```diff
@@ -0,0 +1,158 @@
+"ALERT_ADD" = "Dodaj";
+"ALERT_CANCEL" = "Anuluj";
+"ALERT_DELETE" = "Usuń";
+
+"ADD_STATION_WINDOW_TITLE" = "Dodaj nową stację";
+"EDIT_STATION_WINDOW_TITLE" = "Edytuj stację";
+
+"EDIT_STATION_WINDOW_NAME_PLACEHOLDER" = "Nazwa";
+"EDIT_STATION_WINDOW_URL_PLACEHOLDER" = "http://ścieżka/stacji";
+"EDIT_STATION_WINDOW_IMAGE_ACCESSIBILITY_LABEL" = "Wybierz okładkę";
+
+"ADD_COLLECTION_WINDOW_TITLE" = "Dodaj nową kolekcję";
+"EDIT_COLLECTION_WINDOW_TITLE" = "Edytuj kolekcję";
+
+"EDIT_COLLECTION_WINDOW_NAME_PLACEHOLDER" = "Moja kolekcja";
+
+"SIDEBAR_CONTEXT_EDIT_COLLECTION" = "Edytuj";
+"SIDEBAR_CONTEXT_DELETE_COLLECTION" = "Usuń kolekcję";
+"SIDEBAR_CONTEXT_SHARE_COLLECTION" = "Udostępnij";
+
+"STATION_CONTEXT_EDIT_COLLECTION" = "Edytuj";
+"STATION_CONTEXT_DELETE_COLLECTION" = "Usuń stację";
+"STATION_CONTEXT_MOVE" = "Umieść w kolekcji";
+
+"CONTEXT_ALERT_DELETE_COLLECTION_TITLE" = "Usuń \"%@\"";
+"CONTEXT_ALERT_DELETE_COLLECTION_BODY" = "Czy na pewno chcesz usunąć tę kolekcję i wszystkie powiązane stacje?";
+
+"CONTEXT_ALERT_DELETE_STATION_TITLE" = "Usuń \"%@\"";
+"CONTEXT_ALERT_DELETE_STATION_BODY" = "Czy na pewno chcesz usunąć tę stację?";
+
+"COLLECTION_VIEW_EMPTY_SECTION_TITLE" = "Brak stacji";
+
+"WELCOME_COPY" = "Witamy w Broadcasts! Dodaliśmy już kilka irlandzkich stacji radiowych, aby Twoja biblioteka nie była pusta.\n\nJeżeli wolisz, możesz ręcznie dodawać własne stacje lub wybrać którąś z tysięcy stacji z naszego katalogu.";
+"WELCOME_UPSELL" = "Witamy w Broadcasts! W tej darmowej wersji dodaliśmy już kilka irlandzkich stacji radiowych, aby Twoja biblioteka nie była pusta.\n\nJeżeli przejdziesz na płatną wersję, możliwe będzie ręczne dodawanie własnych stacji lub wybranie którejś z tysięcy stacji z naszego katalogu.";
+"WELCOME_ADD_PRESETS" = "Dodaj domyślne";
+"WELCOME_NO_THANKS" = "Nie, dziękuję";
+
+"GENERATED_COLLECTION_TITLE" = "Moja kolekcja";
+
+"UPSELL_COPY" = "Dzięki za korzystanie z Broadcasts!\n\nJeżeli chcesz, możesz odblokować zaawansowane funkcje za pomocą mikropłatności w aplikacji.\n\nA jeśli wolisz, możesz również dalej korzystać z domyślnych stacji i je edytować!";
+"UPSELL_BUY" = "Kup (%@)";
+"UPSELL_NOT_AVAILABLE" = "Zakup niedostępny";
+"UPSELL_RESTORE" = "Odnów zakup";
+"UPSELL_NO_THANKS" = "Nie, dziękuję";
+
+"TOOLBAR_NEW_CATEGORY_TOOLTIP" = "Nowa kolekcja";
+"TOOLBAR_ADD_STATION_TOOLTIP" = "Dodaj stację";
+"TOOLBAR_SHOW_SOURCES_TOOLTIP" = "Pokaż kolekcje";
+"TOOLBAR_TOGGLE_SIDEBAR_TOOLTIP" = "Ukryj/pokaż pasek boczny";
+"TOOLBAR_TOGGLE_VIEW_TOOLTIP" = "Przełącz widok";
+
+"IMAGE_WELL_REMOVE" = "Usuń";
+"IMAGE_WELL_PASTE" = "Wklej";
+"IMAGE_WELL_COPY" = "Kopiuj";
+
+"CARPLAY_EMPTY_LIBRARY" = "Brak kolekcji";
+"CARPLAY_ALL_STATIONS" = "Wszystkie stacje";
+
+"MAC_ADD_STATION_BUTTON" = "Dodaje stację";
+"MAC_EDIT_STATION_BUTTON" = "Edytuj stację";
+
+"MAC_ADD_COLLECTION_BUTTON" = "Dodaj kolekcję";
+"MAC_EDIT_COLLECTION_BUTTON" = "Edytuj kolekcję";
+
+"STATION_DIRECTORY_WINDOW_TITLE" = "Przeglądaj stacje"
```

Comment:
Tiny change, missing a terminating ;

Reply:
oh man I must've deleted that by accident